src/st/class-wpml-page-builders-app.php

```diff
@@ -25,7 +25,7 @@ public function add_hooks() {
 	}
 
 	public function load_integration() {
-		if ( ! class_exists( 'WPML_ST_Package_Factory' ) ) {
+		if ( ! class_exists( 'WPML_ST_Package_Factory' ) || ! defined( 'WPML_TM_VERSION' ) ) {
```

Comment:
@strategio I don't think this is the right fix. We should still be able to create string packages without TM. The check you should make is where we are integrating the TM functions themselves.

Reply:
@brucepearson, I think I see your point. In the case a user works with Core and ST, create some posts. Later activate TM and tries to translate immediately...

OK, I will close this one!